test/unit/gateways/authorize_net_cim_test.rb

```diff
@@ -12,15 +12,17 @@ def setup
     @credit_card = credit_card
     @address = address
     @customer_profile_id = '3187'
+    @email = 'johndoe@test.com'
```

Comment:
Do you truly want these in all tests? If not, I'd modify the tests themselves where you want this. The fact you've got optional modifications at https://github.com/activemerchant/active_merchant/pull/2690/files#diff-068964b7f2d4b4c8f35e220595c2ee62R25 makes me think this shouldn't be a global change.